controller/src/beerocks/master/beerocks_master_mrouter.cpp

```diff
@@ -52,6 +52,7 @@ bool master_mrouter::init()
     return (bus_subscribe(std::vector<ieee1905_1::eMessageType>{
         ieee1905_1::eMessageType::VENDOR_SPECIFIC_MESSAGE,
         ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_SEARCH_MESSAGE,
+        ieee1905_1::eMessageType::CHANNEL_PREFERENCE_REPORT_MESSAGE,
```

Comment:
please squash this to the previous commit.

Reply:
why? I try to separate commits to atomic units.
I did such a thing in my last PR that merged, and you accepted it then, so why no now?

Reply:
See here - https://github.com/prplfoundation/prplMesh/pull/66#issuecomment-508678164

Reply:
and squash it

Reply:
fixed